atmo/stats/models.py

```diff
@@ -0,0 +1,55 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+from django.contrib.postgres.fields import JSONField
+from django.core.serializers.json import DjangoJSONEncoder
+from django.db import models
+from django.utils import timezone
+
+
+class Metric(models.Model):
+    created_at = models.DateTimeField(
+        editable=False,
+        blank=True,
+        default=timezone.now,
+        db_index=True,
+    )
+    key = models.CharField(
+        max_length=100,
+        db_index=True,
+        help_text='Name of the metric being recorded',
+    )
```

Comment:
I think it might make more sense to create a composite index on the key + created_at timestamp instead of creating individual column indexes.

https://dba.stackexchange.com/questions/27481/is-a-composite-index-also-good-for-queries-on-the-first-field

Or maybe just not bothering with indexing at all.